third_party/rust/adler/v1/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
For this one, you can refer to the copy in https://github.com/brave/brave-core/blob/master/common/licenses/Apache-2.0.

Unlike the `MIT` and `BSD-*` licenses where every author customizes it, the `Apache-2.0` (and `MPL-2.0`) license text is always the same and so we can reuse the same copy of it.

Reply:
Added link to the common license file in https://github.com/brave/brave-core/pull/23822/commits/9c8f0be5336cdf725eec96c5f66ce78265cde498 where I could. Some of the crates are needed by the build but for some reason aren't considered as dependencies by the gnrt script, so I can't override the metadata for them. I left the LICENSE files next to README.chromium files for those.